src/theme/SearchBar/lib/lunar-search.js

```diff
@@ -16,6 +16,13 @@ class LunrSearchAdapter {
             query.term(tokens, {
                 wildcard: lunr.Query.wildcard.TRAILING
             });
+
+            if(versions) {
+                query.term(versions.join(" "), {
+                    fields: ["version"],
+                    boost: 0,
```

Comment:
What is boost 0?

Reply:
> In multi-term searches, a single term may be important than others. For these cases Lunr supports term level boosts. Any document that matches a boosted term will get a higher relevance score, and appear higher up in the results.

https://lunrjs.com/guides/searching.html#boosts

I have lost the code that removes results with a score of 0 that occur when the docs are versioned and just the version matches. This Is the reason to set 0 as "weight" of version term.

I have added the code after these lines.